projects/ngrx.io/content/examples/store-walkthrough/src/app/tests/app.component.1.spec.ts

```diff
@@ -96,3 +97,93 @@ describe('AppComponent', () => {
     //#enddocregion mockSelector
   });
 });
+
+//#docregion resetMockSelector
+describe('AppComponent reset selectors', () => {
+  let store: MockStore<AppState>;
+
+  afterEach(() => {
+    store.resetSelectors();
+  });
+
+  it('should return the mocked value', (done: any) => {
+    TestBed.configureTestingModule({
+      providers: [
+        provideMockStore({
+          initialState: [
+            {
+              id: 'firstId',
+              volumeInfo: {
+                title: 'Initial Title',
+                authors: ['Initial Author'],
+              },
+            },
+          ],
```

Comment:
When mocking selectors, the state doesn't matter.
So, I would prefer to leave it out.

```suggestion
```